From: Daniele Ricci <daniele@casaricci.it>
Date: Fri, 12 Sep 2025 11:38:46 +0200
Subject: [PATCH] Workaround for wrong list divider margin (#119895)

---
 lib/src/list_section.dart | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/lib/src/list_section.dart
+++ b/lib/src/list_section.dart
@@ -90,7 +90,7 @@
 
 // Additional margin of divider used in inset grouped version of list section
 // when there is no leading widgets. Estimated from iOS 14.4 SDK Notes app.
-const double _kInsetAdditionalDividerMarginWithoutLeading = 14.0;
+const double _kInsetAdditionalDividerMarginWithoutLeading = 6.0;
 
 // Color of header and footer text in edge-to-edge variant.
 const Color _kHeaderFooterColor = CupertinoDynamicColor(
